specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Usages.json

```
@@ -143,13 +143,11 @@
           }
         },
         "currentValue": {
-          "type": "integer",
-          "format": "int32",
+          "format": "float",
```

Need to specify type
```suggestion
          "type": "number",
          "format": "float",
```